net.redeclipse.RedEclipse-Legacy.appdata.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>net.redeclipse.RedEclipse-Legacy</id>
+  <metadata_license>CC-BY-SA-3.0+</metadata_license>
+  <project_license>Zlib and CC-BY-SA-3.0+</project_license>
+  <provides>
+    <id>redeclipse.desktop</id>
+  </provides>
+  <launchable type="desktop-id">net.redeclipse.RedEclipse-Legacy.desktop</launchable>
+  <name>Red Eclipse Legacy</name>
+  <summary>First-person shooter with agile gameplay and built-in editor</summary>
+  <description>
+    <p>
+      Red Eclipse is a fun-filled new take on the first person arena shooter,
+      built as a total conversion of Cube Engine 2, which lends itself toward
+      a balanced gameplay, with a general theme of agility in a variety of
+      environments. This is version 1.6.0 of the game, the last version before
+      its development went to a different direction in terms of graphics,
+      gameplay mechanics and other aspects.
+    </p>
+    <ul>
+      <li>Cross-platform multiplayer</li>
+      <li>Features parkour, impulse boosts, dashing, and other tricks</li>
+      <li>Favourite gamemodes with an array of mutators and variables</li>
+      <li>Builtin editor lets you create your own maps cooperatively online</li>
+    </ul>
+  </description>
+  <url type="homepage">https://redeclipse.net/</url>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://redeclipse.net/bits/images/053.jpg</image>
+      <caption>Posing with fire on the map venus</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://redeclipse.net/bits/images/045.jpg</image>
+      <caption>Mine fireworks on the map vault</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://redeclipse.net/bits/images/019.jpg</image>
+      <caption>Overlooking friendly fire on the map cutec</caption>
+    </screenshot>
+  </screenshots>
+  <developer_name>Red Eclipse Team</developer_name>
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-cartoon">none</content_attribute>
```

Comment:
you can omit the none values

Reply:
Sorry, I didn't really understand. Should I delete just " >none</ " , the entire line, or all the lines containing the none value? I think I need suggested change example.